mllib/src/main/scala/org/apache/spark/mllib/fpm/FPGrowth.scala

```diff
@@ -116,7 +116,7 @@ object FPGrowthModel extends Loader[FPGrowthModel[_]] {
         StructField("freq", LongType))
       val schema = StructType(fields)
       val rowDataRDD = model.freqItemsets.map { x =>
-        Row(x.items, x.freq)
+        Row(x.items.toSeq, x.freq)
```

Comment:
We need. This is a special case, `FPGrowthModel` has a type parameter and we use `FPGrowthModel[_]` here. So `x.items` returns `Object[]` instead of `T[]` as we expected and doesn't match the schema.